staging/src/k8s.io/apiserver/pkg/admission/configuration/mutating_webhook_manager_test.go

```diff
@@ -85,6 +88,26 @@ func (f *fakeMutatingWebhookConfigLister) Get(name string) (*v1beta1.MutatingWeb
 	panic("unsupported")
 }
 
+func newFakeMutatingWebhookConfigurationManager(f admissionregistrationinformers.MutatingWebhookConfigurationInformer) generic.Source {
```

Comment:
these really look so similar. Any reason not to unify the tests? Does it work?

Reply:
Actually, I have not found a nice way to do unify.